portals/admin/src/main/webapp/source/src/app/components/APISettings/ListApis.jsx

```diff
@@ -0,0 +1,275 @@
+/*
+* Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
* Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Fixed in https://github.com/wso2/apim-apps/pull/519/commits/426260ecaebe104033a2bdebe2ec8a03eef3d248